[AST] Bump 7.4 and remove unnecessary CO from overheal

otocephaly opened the pull request:
bumped 7.4 and removed the unnecessary status from overhealing for collective unconsciousness

## Pull request type

- [ ] This is new functionality or an addition to existing functionality
- [x] This is a bugfix to existing functionality
- [x] This is a patch bump

## Pull request details

<!-- If this PR is for new functionality or a bugfix, please complete the section below.  For a patch bump, delete this comment and the section below.  The context for your change is important to help the reviewer understand what the change is supposed to do - please provide an appropriate link or description to help the review process. -->
- [x] The goal of this PR is detailed below:

		*[For new functionality / added functionality, describe what is being added]*
		*[For bugfixes, please provide reproduction steps and what the expected vs actual behavior is in production right now]*
    - not count the non-heal, non-mit portion of collective unconscious and bump
    

<!-- If this PR is for a patch bump, please complete the section below.  For functionality additions or bug fixes, delete this comment and the section below -->
- [x] The job(s) in this patch bump had no changes for this patch
    - the change to Collective Unconscious has no additional effect on analysis since it just changed one status from 5 to 10 seconds. The other change was just to remove the non-heal, non-mit portion from being counted unnecessarily in overheal.

## Testing / Validation

<!-- If this PR contains any new functionality or bugfixes, please include log links that reviewers can use to help verify your changes.  Reviewers may also find additional logs to check your PR with, but having initial logs is important to speed the review process, especially for corner cases or hard to trigger analysis flags.  If this PR is a patch bump with no changes or potency only changes, you can delete this section. -->
- [x] I used the log(s) listed below to develop and test this bugfix:
- Any log where an AST cast collective unconscious at least once will show this. I used this one as an example https://www.fflogs.com/reports/t9X8rHvczjymMTgq?fight=3&source=6

   *[Provide a list of log links - either direct to FFLogs or the xivanalysis log link for development or production]*

## Job Maintenance
- [x] I have collaborated with one or more job maintainers for the job(s) in this PR while developing it

## Screenshots
Before
<img width="1038" height="1030" alt="image" src="https://github.com/user-attachments/assets/063d9b1d-a696-40f1-86f7-23e2fe95b7e3" />

After
<img width="1342" height="1109" alt="image" src="https://github.com/user-attachments/assets/01a4d345-6bb4-4572-8a7b-a517f428cf50" />

otocephaly committed:
[AST] Bump 7.4 and remove unnecessary CO from overheal

bumped 7.4 and removed the unnecessary status from overhealing for collective unconsciousness

otocephaly committed:
fixed mitigation time

used to be 5 seconds, now is 10 seconds
old duration wasn't updated since a duration was attached

AkairyuGestalter reviewed (changes requested):
The adjustment to the CU status duration should probably be in the patch 7.4 status layer file rather than in root. If you rebase this onto the current dawntrail branch, it should be there, and that should also fix the checks error now that the 7.4 patch is actually defined too.

otocephaly commented:
> The adjustment to the CU status duration should probably be in the patch 7.4 status layer file rather than in root. If you rebase this onto the current dawntrail branch, it should be there, and that should also fix the checks error now that the 7.4 patch is actually defined too.

oop you are correct -- I wasn't thinking
I don't recall when it was originally bumped to 5s, so I'll add it into there as well

otocephaly committed:
fixed duration for collective mit

added mit into patch layer and corrected status duration to match 7.0 original duration

AkairyuGestalter commented on src/parser/jobs/ast/modules/Overheal.tsx:
```diff
@@ -10,6 +11,12 @@ const NEUTRAL_SECT_APPLICATION_BUCKET_ID = 1
 export class Overheal extends CoreOverheal {
 	protected override checklistDisplayOrder = DISPLAY_ORDER.OVERHEAL_CHECKLIST
 
+	private ignoredStatuses: Array<Status['id']> = [
```
Rather than special-case handling this in considerHeal, you might want to look at how I handled this for Kardia in SGE. If you make a trackedHealCategories bucket for it, and set that bucket's ignore prop to false, the rest is taken care of for you by the core module.

otocephaly replied:
appreciate the heads up and I do like consistency
(I probably should've checked other jobs but I wasn't thinking lol it's just one of those days/weeks/months/years/decades/lifetimes)
I've updated that in the most recent commit
Thank you!

AkairyuGestalter reviewed (commented):
One more comment that you can take or leave, but seems like would be a simple fix.

otocephaly committed:
updated to be consistent

updated to be consistent with other job's application of similar actions (e.g. SGE Kardia)

otocephaly committed:
oops space

oops space
